### PR TITLE
Added JamunaTV bd.m3u #bd for bangladesh

### PR DESCRIPTION
**Pull request by f4rh4d-4hmed**
Jamuna Television has been added. Works on national level network.

### CODE REVIEW

File: streams/bd.m3u [hunk 1]
```diff
@@ -9,3 +9,5 @@ https://amigofx.com:1936/channelsreporter/channelsreporter/playlist.m3u8
 https://iptvbd.live/test/1080.m3u8
 #EXTINF:-1 tvg-id="EkusheyTV.bd",Ekushey TV (480p)
 https://ekusheyserver.com/etvlivesn.m3u8
+#EXTINF:-1 tvg-id="JamunaTelevision.bd",Jamuna TV (720p)
```
Comment (Dum4G) [inline streams/bd.m3u:12]:
```suggestion
#EXTINF:-1 tvg-id="JamunaTV.bd",Jamuna TV (720p)
```

**Comment by Dum4G** [comment]
@f4rh4d-4hmed connection times out, can you confirm it still work for you?

**Comment by f4rh4d-4hmed** [comment]
Yes, its working fine for me. As its a local server it will only work for Bangladeshi people connected to broadband network. I wish for Bangladeshi and some indian to access this free IPTV.
Screenshot_20240804-145518_Telegram

File: streams/bd.m3u [hunk 2]
```diff
@@ -9,3 +9,5 @@ https://amigofx.com:1936/channelsreporter/channelsreporter/playlist.m3u8
 https://iptvbd.live/test/1080.m3u8
 #EXTINF:-1 tvg-id="EkusheyTV.bd",Ekushey TV (480p)
 https://ekusheyserver.com/etvlivesn.m3u8
+#EXTINF:-1 tvg-id="JamunaTV.bd",Jamuna TV (720p)
```
Comment (Dum4G) [inline streams/bd.m3u:12]:
```suggestion
#EXTINF:-1 tvg-id="JamunaTV.bd",Jamuna TV (720p) [Geo-blocked]
```

**Review by Dum4G — APPROVED** [review]

**Review by f4rh4d-4hmed — COMMENTED** [review]
Thanks

**Review by Sphinxroot — APPROVED** [review]

**Pull request merged**